pegasus/src/homepage.rb

```diff
@@ -127,7 +127,7 @@ def self.get_actions(request)
     end
 
     hoc_mode = DCDO.get('hoc_mode', CDO.default_hoc_mode)
-    if code_break_takeover
+    if code_break_takeover && request.language == "en"
```

Comment:
I think we'd still have a problem [here](https://github.com/code-dot-org/code-dot-org/pull/34198/files#diff-b64653aca2893bcd9da66a45ac6e6c83R117), and should therefore check for `en` [here](https://github.com/code-dot-org/code-dot-org/pull/34198/files#diff-b64653aca2893bcd9da66a45ac6e6c83L108) instead, so that `code_break_takeover` only gets set for `en` users.